host.go

```diff
@@ -305,7 +306,44 @@ func (h *gpbftHost) ReceiveDecision(decision *gpbft.Justification) time.Time {
 		return time.Now().Add(h.manifest.ECDelay)
 	}
 
-	return ts.Timestamp().Add(h.manifest.ECDelay)
+	if decision.Vote.Value.HasSuffix() {
+		// we decided on something new, use just the ECDelay
+		return ts.Timestamp().Add(h.manifest.ECDelay)
+	}
+
+	// we decided on base, calculate how much we should back off
+	// all of this should go into manifest but I think Alfonso dislikes me already :P
+	const (
+		minBackoff = 2.
+		maxBackoff = 20. // 10m with 30s ECDelay
+	)
+	// the backoff is defined in multiples of ECDelay starting at the last finalized tipset
+	// each additional base decision beyond that will incurr the maxBackoff
+	var backoffTable = []float64{2, 2.3, 2.69, 3.197, 3.8561, 4.71293, 4.71293, 5.826809, 7.2748517, 9.15730721} // 1.3^i+1 backoff, table for more flexibility
```

Comment:
This probably should have more elements if we reduced the exponent to 1.3 for a smoother transition.